Resources/Prototypes/Entities/Structures/Machines/pitching_machine.yml

```diff
@@ -0,0 +1,60 @@
+- type: entity
+  id: PitchingMachine
+  name: pitching machine
+  description: Hey batter batter! Swing!
```

Comment:
This prototype can be made way shorter by inheriting `BaseMachinePowered`

Reply:
I'd need to overwrite basically all of basemachinepowered if i did